active/tab/tabcontrol.md

```diff
@@ -0,0 +1,580 @@
+
+# Examples
+<!-- Use this section to explain the features of the API, showing
+example code with each description. The general format is: 
+  feature explanation,
+  example code
+  feature explanation,
+  example code
+  etc.-->
+
+<!-- As an example of this section, see the Examples section for the PasswordBox control 
+(https://docs.microsoft.com/windows/uwp/design/controls-and-patterns/password-box#examples). -->
+
+## To replicate the behavior of Microsoft Edge
+
+``` xml
+<TabView TabViewItemWidthMode="Equal"
+            CanCloseTabs="True"
+            CloseButtonOverlay="OnHover"
+            CanDragTabs="True"
+            CanReorderTabs="True"
+            TabDraggedOutside="OpenTabInNewWindow"
+            AddTabButtonClick="NewTab_Click">
+    ...
+</TabView>
+```
+
+## Put TabView in the titlebar area
+
+![A sample app that shows how TabView can extend into the titlebar area](images/tab-extend-to-title.png)
+
+This sample demonstrates how to extend the TabView into the title bar area and also specify a portion of the UI as the draggable region, using [CoreApplicationViewTitleBar.ExtendViewIntoTitleBar](https://docs.microsoft.com/uwp/api/Windows.ApplicationModel.Core.CoreApplicationViewTitleBar.ExtendViewIntoTitleBar). Per the [titlebar customization guidelines](https://docs.microsoft.com/en-us/windows/uwp/design/shell/title-bar), we must set a [specific drag region](https://docs.microsoft.com/en-us/windows/uwp/design/shell/title-bar#draggable-regions) (the space that a user can use to drag the window around the screen using mouse or touch). If we don't specify the drag region, the entire titlebar area remains draggable (meaning input won't be routed to the tabs, making the tabs unclickable).
+
+``` xml
+<Page>
+    <Grid>
+        <TabView>
+            <TabViewItem Icon="Home" Header="Home" IsCloseable="False" />
+            <TabViewItem Icon="Document" Header="Document 1" />
+            <TabViewItem Icon="Document" Header="Document 2" />
+            <TabViewItem Icon="Document" Header="Document 3" />
+        </TabView>
+
+        <Grid x:Name="CustomDragRegion" Width="200" Height="40" HorizontalAlignment="Right" VerticalAlignment="Top" />
+    </Grid>
+</Page>
+```
+
+``` csharp
+// Customize the titlebar area using the guidance from here: https://docs.microsoft.com/en-us/windows/uwp/design/shell/title-bar
+public MainPage()
+{
+    this.InitializeComponent();
+
+    var coreTitleBar = CoreApplication.GetCurrentView().TitleBar;
+    coreTitleBar.ExtendViewIntoTitleBar = true;
+
+    Window.Current.SetTitleBar(CustomDragRegion);
+}
+```
+
+## Create a new window when tearing out tabs
+
+![An example picture showing that TabView can be torn out and moved to its own window.](images/tab-new-window.png)
+
+See the [TabView tear out sample](https://github.com/windows-toolkit/Sample-TabView-TearOff/tree/master/TabViewTear) for a more complete sample.
+
+``` xml
+<TabView CanDragTabs="True"
+            CanReorderTabs="True"
```

Comment:
There is no TabReordered event. How does the app know that the user reordered the tabs?